tests/c/intrinsic_noinline.c

```diff
@@ -0,0 +1,59 @@
+// Run-time:
+//   env-var: YKD_PRINT_IR=jit-pre-opt
+//   env-var: YKD_SERIALISE_COMPILATION=1
+//   env-var: YKD_PRINT_JITSTATE=1
+//   stderr:
+//     jit-state: start-tracing
+//     jit-state: stop-tracing
+//     --- Begin jit-pre-opt ---
+//     ...
+//     define i8 @__yk_compiled_trace_0(ptr %0, ptr %1, i64 %2...
+//        ...
+//        call void @llvm.memcpy...
+//        ...
+//     }
+//     ...
+//     --- End jit-pre-opt ---
+//     jit-state: enter-jit-code
+//     ...
+//     jit-state: enter-stopgap
+//     ...
+//   stdout:
+//     3
+
+// Check that basic trace compilation works.
```

Comment:
Update this comment.

Reply:
Fixed in [ff466a4](https://github.com/ykjit/yk/pull/570/commits/ff466a4ac9727fb8ea5d8b3f9c2beea57b63b526)